src/remodel_api/remodel.rs

```diff
@@ -369,7 +499,12 @@ impl Remodel {
         }
 
         if response.status().is_success() {
-            Ok(())
+            let asset_id = response.headers().get("roblox-assetid");
+
+            match asset_id {
+                Some(asset_id) => Ok(String::from(asset_id.to_str().unwrap())),
```

Comment:
I don't know if we can unwrap here. We don't control this value, so whatever can cause `to_str` to not return a value might fail!